miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1327,13 +1321,7 @@ private static QcTypeDto asDto(QcType from) {
 
   private static QcType to(QcTypeDto from) {
     QcType to = new QcType();
-    to.setQcTypeId(from.getId());
-    to.setName(from.getName());
-    to.setDescription(from.getDescription());
-    to.setQcTarget(from.getQcTarget());
-    to.setUnits(from.getUnits());
-    to.setPrecisionAfterDecimal(from.getPrecisionAfterDecimal());
-    to.setArchived(from.isArchived());
```

Comment:
All fields on the DTO should still be included in the conversion method for consistency and predictability